client.js

```diff
@@ -60,14 +68,24 @@ const isValidURL = function (url) {
   return false;
 };
 
-//GOAL:If a message does not pass validation, it should be sent to the DLQ and removed from the input topic.
+const sendMsg = (message, token) => {
+  axios
+    .post(
+      `https://${URL}/message`,
+      { message },
+      { headers: { "X-Gotify-Key": token } }
+    )
+    .then(console.log("sendMsg", message))
```

Comment:
Last nitpick, this log message should also be changed:
```suggestion
    .then(console.log("Sending message:", message))
```

In the future, we can hide this behind a `LOG_LEVEL` flag, as it will be overly verbose during regular usage.

Reply:
Just a callout that this log message still needs to be updated.

Reply:
Log message updated.